Code/lib/motor/motor.cpp

```diff
@@ -0,0 +1,81 @@
+#include "Arduino.h"
+#include "motor.h"
+
+Drive::Drive(MotorPins motorpins, int turndelay, int turnspeed)
+{
+    motor_pins = motorpins;
+    turn_delay = turndelay;
+    turn_speed = turnspeed;
+
+    // Set all the motor control pins to outputs
+    pinMode(motor_pins.ENA, OUTPUT);
+    pinMode(motor_pins.ENB, OUTPUT);
+    pinMode(motor_pins.IN1, OUTPUT);
+    pinMode(motor_pins.IN2, OUTPUT);
+    pinMode(motor_pins.IN3, OUTPUT);
+    pinMode(motor_pins.IN4, OUTPUT);
```

Comment:
Can you move this to an initialize() function that can be called in the setup function and the initial class initialization can occur in globally outside the setup function